config/dev.exs

```diff
@@ -61,7 +61,7 @@ config :phoenix, :plug_init_mode, :runtime
 # Configure your database
 config :retro, Xrt.Repo,
   username: System.get_env("DB_USER") || "postgres",
-  password: System.get_env("DB_PASSWORD") || "",
+  password: System.get_env("DB_PASSWORD") || "password",
```

Comment:
default password is postgres

Reply:
last minute changes gone wrong 😅 changed!